Ports/halflife/package.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port="half-life"
```

Comment:
Should probably match the directory name, even though it isn't enforced by the linter (yet).